test/parallel/test-worker-workerdata-sharedarraybuffer.js

```diff
@@ -0,0 +1,32 @@
+// Flags: --expose-gc --experimental-worker
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const { Worker } = require('worker_threads');
+
+{
+  const sharedArrayBuffer = new SharedArrayBuffer(12);
+  const local = Buffer.from(sharedArrayBuffer);
+
+  const w = new Worker(`
+    const { parentPort, workerData } = require('worker_threads');
+    const local = Buffer.from(workerData.sharedArrayBuffer);
+
+    parentPort.on('message', () => {
+      local.write('world!', 6);
+      parentPort.postMessage('written!');
+    });
+  `, {
+    eval: true,
+    workerData: { sharedArrayBuffer }
+  });
+  w.on('message', common.mustCall(() => {
+    assert.strictEqual(local.toString(), 'Hello world!');
+    global.gc();
+    w.terminate();
+  }));
+  w.postMessage({ sharedArrayBuffer });
```

Comment:
Do we need to pass in `sharedArrayBuffer` here? It doesn’t seem to be consumed in the worker.

Reply:
cc @Fishrock123